testutil/client.go

```diff
@@ -258,13 +258,16 @@ type LoginParams struct {
 // and returns the access JWT and refresh JWT extracted from
 // the HTTP response
 func HttpLogin(params *LoginParams) (*HttpToken, error) {
+	fmt.Println("HttpLogin: params data ", params)
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein

note: need to rebase